wallet/src/de/schildbach/wallet/ui/main/WalletTransactionsFragment.java

```diff
@@ -465,154 +198,6 @@ private void showEmptyView() {
-            org.bitcoinj.core.Context.propagate(Constants.CONTEXT);
-
-            final Set<Transaction> transactions = wallet.getTransactions(true);
-            final List<Transaction> filteredTransactions = new ArrayList<Transaction>(transactions.size());
-
-            for (final Transaction tx : transactions) {
-                final boolean sent = tx.getValue(wallet).signum() < 0;
-                final boolean isInternal = tx.getPurpose() == Purpose.KEY_ROTATION;
-
-                if ((direction == Direction.RECEIVED && !sent && !isInternal) || direction == null
-                        || (direction == Direction.SENT && sent && !isInternal))
-                    filteredTransactions.add(tx);
-            }
-
-            Collections.sort(filteredTransactions, TRANSACTION_COMPARATOR);
-
-            return filteredTransactions;
-        }
-
-        private final ThrottlingWalletChangeListener transactionAddRemoveListener = new ThrottlingWalletChangeListener(
-                THROTTLE_MS, true, true, false) {
-            @Override
-            public void onThrottledWalletChanged() {
-                safeForceLoad();
-            }
-        };
-
-        private final BroadcastReceiver walletChangeReceiver = new BroadcastReceiver() {
-            @Override
-            public void onReceive(final Context context, final Intent intent) {
-                safeForceLoad();
-            }
-        };
-
-        private void safeForceLoad() {
-            try {
-                forceLoad();
-            } catch (final RejectedExecutionException x) {
-                log.info("rejected execution: " + TransactionsLoader.this.toString());
-            }
-        }
-
-        private static final Comparator<Transaction> TRANSACTION_COMPARATOR = new Comparator<Transaction>() {
-            @Override
-            public int compare(final Transaction tx1, final Transaction tx2) {
-                final boolean pending1 = tx1.getConfidence().getConfidenceType() == ConfidenceType.PENDING;
-                final boolean pending2 = tx2.getConfidence().getConfidenceType() == ConfidenceType.PENDING;
-
-                if (pending1 != pending2)
-                    return pending1 ? -1 : 1;
-
-                final Date updateTime1 = tx1.getUpdateTime();
-                final long time1 = updateTime1 != null ? updateTime1.getTime() : 0;
-                final Date updateTime2 = tx2.getUpdateTime();
-                final long time2 = updateTime2 != null ? updateTime2.getTime() : 0;
-
-                if (time1 != time2)
-                    return time1 > time2 ? -1 : 1;
-
-                return tx1.getHash().compareTo(tx2.getHash());
-            }
-        };
-
-    }
-
-    @Override
-    public void onSharedPreferenceChanged(final SharedPreferences sharedPreferences, final String key) {
-        if (Configuration.PREFS_KEY_BTC_PRECISION.equals(key) || Configuration.PREFS_KEY_REMIND_BACKUP.equals(key) ||
```

Comment:
`PREFS_KEY_BTC_PRECISION` never changes. The rest is moved to the `WalletActivity` since it's used for updating the shortcuts in there.